subsys/tracing/user/tracing_user.c

```diff
@@ -25,6 +25,55 @@ void __weak sys_trace_isr_exit_user(void) {}
 void __weak sys_trace_idle_user(void) {}
 void __weak sys_trace_sys_init_enter_user(const struct init_entry *entry, int level) {}
 void __weak sys_trace_sys_init_exit_user(const struct init_entry *entry, int level, int result) {}
+void __weak sys_trace_gpio_pin_interrupt_configure_enter_user(const struct device *port,
```

Comment:
Is there a way to make the CI not complain about these?

Reply:
Maybe 
```
/* clang-format off */
...
/* clang-format on */
```